frontend/packages/kubevirt-plugin/src/hooks/use-show-error-toggler.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+export const useShowErrorToggler = (
+  initialShowError: boolean = false,
+  initialIsValid: boolean = false,
```

Comment:
nit: if a default is set, the param should be optional

Reply:
default value makes it not optional and `initialIsValid?` cannot be used

Reply:
The default value is applied if the parameter is omitted when calling, so it is optional.
If a parameter is not optional, then its value must be provided (either true or false in that case).
But it's not a blocker, imo.

Reply:
I get that it is actually optional for the caller but not according to typescript (which is probably concerned about the user of the parameter - the function?)